### PR TITLE
Fix Breakage with python 3.10

### PR DESCRIPTION
**Pull request by d-air1**
The main problem seems to be that QPoint does not accept floating point values. It must have at one time otherwise this app wouldn't have worked for so long. For now i have wrapped the floating point numbers in an int() function. It seems to fix the issue as far as what I have tested. I'm not really familiar with Qt, but the values I converted to int seem to correspond to text size and the volume changer. Do let me know if I have messed up the logic of your app here.

### CODE REVIEW

File: cattqt.py [hunk 1]
```diff
@@ -0,0 +1,1227 @@
+# Copyright 2020 - Scott Moreau
```
Comment (soreau) [inline cattqt.py:1]: Did you mean to copy this file? Seems like a mistake.

Comment (d-air1) [reply cattqt.py:1]: I opened a new pull request with the same name that should fix me accidentally copying the whole file. I'll close this one.

**Comment by soreau** [comment]
If you leave this one open, you can force-push to d-air1:fix-breakage-with-python3.10 and it should update, no need to create a new PR.

**Comment by d-air1** [comment]
I've already created the new pr though. Should I still try the force push?

**Comment by soreau** [comment]
Force pushing to the 'from branch' in a PR is the expected way to update it. If each change had a new PR, this would become quite chaotic.

**Comment by d-air1** [comment]
I can't seem to make git use the this branch even though both have the same name.

**Pull request closed without merging**